src/stories/material-description/material-description.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import MaterialLink from "../material-link/material-link";
+
+interface MaterialDescriptionProps {
+  materialDescription?: string;
+}
+
+const MaterialDescription: React.FC<MaterialDescriptionProps> = ({
+  materialDescription,
+}) => {
+  return (
+    <section className="material-description">
+      <h2 className="text-header-h4 pb-24">Beskrivelse</h2>
+      <p className="text-body-large ">{materialDescription}</p>
+      <div className="material-description__links mt-32">{MaterialLinks}</div>
+    </section>
+  );
+};
+
+export default MaterialDescription;
+
+const MaterialLinks = [
```

Comment:
Is there no linting that says that this variable should be defined before it is being used in the MaterialLink component?
I guess it works, but I would have thought that it did not work :)